packages/react-dev-overlay/src/internal/container/Errors.tsx

```diff
@@ -53,6 +53,23 @@ function getErrorSignature(ev: SupportedErrorEvent): string {
   return ''
 }
 
+function makeClickable(text: string): JSX.Element[] {
+  // Regex Checks for http:// or https://
+  const regx = /(https?:\/\/)([^\s/$.?#].[^\s]*)/gi
+
+  return text.split(' ').map((word, index, array) => {
```

Comment:
`.replace` supports regex matching and replacing out of the box, so that's probably a better approach to go with.

Here's an example:

```js
'hello https://example.com/123456'.replace(/https?:\/\/[^\s/$.?#].[^\s]*/, (match) => { return `<a href="${match}">${match}</a>` })
```

Reply:
Ah! Alright. Will replace the split and map with replace

Reply:
This might only work if  I use dangerouslySetInnerHTML, 
To avoid using it, will have to dynamically generate the elements like I was doing previously

Ref: 609a933